lib/project_types/script/layers/infrastructure/rust_project_creator.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Script
+  module Layers
+    module Infrastructure
+      class RustProjectCreator
+        include SmartProperties
+        property! :ctx, accepts: ShopifyCli::Context
+        property! :extension_point, accepts: Domain::ExtensionPoint
+        property! :script_name, accepts: String
+        property! :path_to_project, accepts: String
+
+        ORIGIN_BRANCH = 'main'
+        SAMPLE_PATH = 'default'
+
+        def setup_dependencies
+          git_init
+          setup_remote
+          setup_sparse_checkout
+          pull
+          clean
+        end
+
+        def bootstrap
```

Comment:
@jacobsteves this is now empty and `setup_dependencies` defines everything as per our discussion.